crates/biome_markdown_parser/tests/md_test_suite/ok/thematic_break_block.md.snap

```diff
@@ -19,50 +20,35 @@ ___
 ## AST
 
 ```
-MdDocument {
-    bom_token: missing (optional),
-    value: MdBlockList [
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@0..6 "***" [Whitespace("   ")] [],
-        },
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@6..11 "***" [Newline("\n"), Whitespace(" ")] [],
-        },
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@11..18 "- - -" [Newline("\n"), Whitespace(" ")] [],
-        },
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@18..22 "___" [Newline("\n")] [],
-        },
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@22..30 "_ _ _" [Newline("\n"), Newline("\n"), Whitespace(" ")] [],
-        },
-        MdThematicBreakBlock {
-            value_token: MD_THEMATIC_BREAK_LITERAL@30..37 "* * *" [Newline("\n"), Newline("\n")] [],
-        },
-    ],
-    eof_token: EOF@37..37 "" [] [],
-}
+   ***
+ ***
+ - - -
+___
+
+ _ _ _
+
+* * *
```

Comment:
I believe this is a regression of the changes in the testing suite. Please revert them.